### PR TITLE
Add lucashi fact

### PR DESCRIPTION
**Pull request by tituschow**
pls

### CODE REVIEW

File: assets/lucashifacts [hunk 1]
```diff
@@ -92,6 +92,7 @@ lucashi got his Google Pixel phone for free
 lucashi is allergic to cats
 lucashi cut himself for the first time while cooking when making nachos
 lucashi is a gourmet artisanal grilled cheese chef
+lucashi walked into the fridge door completely sober
```
Comment (Caelink) [inline assets/lucashifacts]: pls add to end of file

**Review by Caelink — CHANGES_REQUESTED** [review]

**Review by Hazematman — CHANGES_REQUESTED** [review]
Needs more details

File: assets/lucashifacts [hunk 2]
```diff
@@ -99,3 +99,4 @@ lucashi is a recognized regular at Chainsaw
 lucashi is the first one to buy a pitcher. Always.
 lucashi was on the most lit floor of UWP
 lucashi procrastinates by watching fallout lore videos on youtube
+lucashi walked into the fridge door completely sober and spilled vegetables over himself
```
Comment (Caelink) [inline assets/lucashifacts]: `on himself` or `all over himself` pls

**Review by Hazematman — CHANGES_REQUESTED** [review]
all over himself*

**Review by Caelink — CHANGES_REQUESTED** [review]

**Review by Caelink — APPROVED** [review]

**Review by Hazematman — APPROVED** [review]

**Pull request merged**